pyproject.toml

```diff
@@ -34,15 +34,17 @@ extend-select = ["I"]
 # Questions commonly use "ambiguous variable names" to represent specific
 # concepts, e.g. I for an identity matrix. We'll allow these to be used in
 # questions.
-"exampleCourse/questions/**/*.py" = ["E741"]
+"exampleCourse/questions/**/*.py" = ["E741", "N806", "N803"]
 # The zygote specifically uses imports in strange places to facilitate the
 # preloading of commonly-used modules.
 "zygote.py" = ["E402", "F401"]
-
+# Ignore uppercase variables (e.g. G for a graph)
+"apps/prairielearn/elements/**/*.py" = ["N806", "N803"]
```

Comment:
I'm actually in favor of enabling these. Capital single letter names could usually be made more descriptive (i.e., I would strongly prefer `graph` to just `G`).